lib/script/script.js

```diff
@@ -866,10 +887,64 @@ Script.buildPublicKeyHashOut = function(to) {
   }
   var s = new Script();
   s.add(Opcode.OP_DUP)
-    .add(Opcode.OP_HASH160)
-    .add(to.hashBuffer)
-    .add(Opcode.OP_EQUALVERIFY)
-    .add(Opcode.OP_CHECKSIG);
+   .add(Opcode.OP_HASH160)
+   .add(to.hashBuffer)
+   .add(Opcode.OP_EQUALVERIFY)
+   .add(Opcode.OP_CHECKSIG);
+  s._network = to.network;
+  return s;
+};
+
+/**
+ * @returns {Script} a new pay to public key hash 256 output for the given
+ * address or public key and optionally a cold staking address
+ * @param {(Address|PublicKey)} to - destination address or public key
+ * @param {(string)} coldStakingAddress - cold staking address
+ */
+Script.buildPublicKeyHashOut256 = function(to, coldStakingAddress) {
```

Comment:
This should be to build a coldstake output, as a normal 256 out wont be the same as this?